src/frontend/src/types/modules.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.module.css" {
+  const classes: { [key: string]: string };
+  export default classes;
+}
```

Comment:
What do you think about using CSS in js with a libray like https://styled-components.com/
Moreover, you plan to use Grommet and this library rely on styled component to create their component and using styles component is the best way to override a grommet component if you want to change locally its styles.